web/src/features/feature-flags/api.ts

```diff
@@ -1,17 +1,13 @@
 import { useQuery } from '@tanstack/react-query';
-import { getBasePath, getHeaders, QUERY_KEYS } from 'api/helpers';
+import { getBasePath, QUERY_KEYS } from 'api/helpers';
 
 import { FeatureFlags } from './types';
 
 export async function getFeatureFlags(): Promise<FeatureFlags> {
   const path: URL = new URL(`/${QUERY_KEYS.FEATURE_FLAGS}`, getBasePath());
-  const requestOptions: RequestInit = {
-    method: 'GET',
-    headers: await getHeaders(path),
-  };
 
   try {
-    const response = await fetch(path, requestOptions);
```

Comment:
This is one option, but would it be better to enable auth in the backend instead of removing it? 🤔

Reply:
No cause that would still trigger options requests, which I want to avoid if it's not needed.

Right now every request we make with custom headers triggers two requests from the browser. One OPTIONS request to see if the action is allowed and then the GET request.

These OPTIONS request are now the primary cause of the spikes in the app-backend cause they cannot be cached by cloudflare.

Reply:
Hmm yeah okay, I see - guess we won't be hurt by DDOS if it's cached on Cloudflare 😅